static/app/components/gridEditable/index.tsx

```diff
@@ -115,7 +115,7 @@ type GridEditableProps<DataRow, ColumnKey> = {
    * - `columnSortBy` is not used at the moment, however it might be better to
    *   move sorting into Grid for performance
    */
-  title?: string;
+  title?: string | ReactNode;
```

Comment:
ReactNode includes string, so don't need both.

```suggestion
  title?: ReactNode;
```